Overview.src.html

```diff
@@ -1498,7 +1498,9 @@ <h3 id="should-response-to-request-be-blocked-due-to-mime-type?"><dfn title="sho
  <li><p>Let <var>type</var> be <var>request</var>'s <span title=concept-request-type>type</span>.
 
  <li><p>If <var>type</var> is "<code title>script</code>", and <var>MIMEType</var> starts with
- `<code title>image/</code>`, then return <b title>blocked</b>.
+ `<code title>audio/</code>`, `<code title>image/</code>` or `<code title>video/</code>`, then return <b title>blocked</b>.
+
+ <li><p>If <var>type</var> is "<code title>script</code>", and <var>MIMEType</var> is `<code title>text/csv</code>`, then return <b title>blocked</b>.
```

Comment:
Nit: I'd suggest combining this with the previous step, as we do in other places. That is, something like:

```
3.  If |type| is `script`, and any of the following statements are true, return `blocked`:

    * |MIMEType| starts with `audio/`, `image/`, or `video/`.
    * |MIMEType| is `text/csv`.
```